lib/flutter_inapp_purchase.dart

```diff
@@ -40,10 +40,10 @@ class FlutterInappPurchase {
   static MethodChannel get channel => _channel;
 
   final Platform _pf;
-  final http.Client? _httpClient;
+  late http.Client _httpClient;
```

Comment:
`_httpClient` will be always assigned on [here](https://github.com/dooboolab/flutter_inapp_purchase/pull/322/files#diff-09ef645a93240c5c6f0a3209c0af6a335d2cdb16bf552e1e5533677bee2e5202R56
).

So, I think it's better to use `late` than using nullable.

Reply:
👍 Reasonable